middleware/groupMiddleware.js

```diff
@@ -0,0 +1,25 @@
+import { bodySchemaForCreate, bodySchemaForUpdate } from '../schemas/groupSchema';
+import ApiError from '../error/ApiError';
+
+
+const createGroupMiddleware = async (req, res, next) => {
+    try {
+        await bodySchemaForCreate.validateAsync(req.body);
+
+        return next();
+    } catch (err) {
+        res.status(400).json(ApiError.badRequest(err.details[0].message));

```

Comment:
It will be nicer if you will get status code `400` from `badRequest` API error

Reply:
Fixed